### ISSUE TITLE
with-strict-csp is incompatible with static pages

### ISSUE THREAD

**Issue by UncleClapton** [vercel/next.js#10472]
# Examples bug report

## Example name

with-strict-csp

## Describe the bug

When attempting to load a statically generated page with strict csp, `_document.js` will error upon attempting to access the request object. In the case of the `with-strict-csp` example, that error is:
```
TypeError: Cannot read property 'locals' of undefined
    at Function.getInitialProps ...
```

## To Reproduce

1. Create with-strict-csp example app using `create-next-app`
2. Run server and access page.
3. Get error.

## Expected behavior

The page to load and display the Hello World text.

## System information

- OS: Windows 10 2004 (insiders)
- Browser: N/A
- Version of Next.js: 9.0.0+

## Additional context

Adding `getInitialProps` to the page or to a custom `_app.js` will fix this issue. I understand if this turns out to be a limitation of static pages and not a bug, but at the very least I think the example should be updated.

**Comment by UncleClapton** [comment]
I just noticed this was fixed with #13922 

I'll close this out.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
